src/util/cilUtil.ml

```diff
@@ -0,0 +1,43 @@
+(** Functionality to create [VarQuery.t] lists from a list of names of global variable and function definitions *)
+(* Separate file from cilfacade.ml, for functions that depend both on [Node] and [VarQuery]. Avoids dependency cycle *)
```

Comment:
Couldn't all of this just go into `VarQuery` itself?
`CilUtil` is quite confusing considering there's also `Cilfacade` in `src/util/`.